modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -102,7 +102,7 @@ const initModel = (editor: Editor) => {
   const Model = ModelManager.get(model);
 
   try {
-    editor.model = Model(editor, modelUrl) || {};
+    editor.model = Model(editor, modelUrl) as Model;
```

Comment:
This is incorrect as `Model` can return `undefined` (void) based on the current types so the old version was correct. If this is mandatory then you're going to have to fix other things.

- [ ] Revert or fix types so a return type is required

Reply:
If RTC doesn't provide a model and the DOM model fails to initialise, the whole editor should fail. In most respects `editor.model` is designed to be the same as `editor.theme`.

Reply:
Yep, it has been reverted so should be good now.